packages/rx_bloc_list/lib/src/extensions/identifiable_extensions.dart

```diff
@@ -109,27 +108,29 @@ extension ModelManageEvents<E extends Identifiable> on Stream<E> {
       _withLatestFromList(list).flatMap((tuple) async* {
         final identifiableInList = tuple.list
             .firstWhereOrNull((element) => element.id == tuple.item.id);
-
-        switch (await operationCallback(tuple.item, identifiableInList)) {
+        switch (await operationCallback(tuple.item)) {
           case ManageOperation.merge:
             yield ManagedList(
-              tuple.list._mergeWithList([tuple.item]),
+              tuple.list.mergeWithList([tuple.item]),
               operation: ManageOperation.merge,
               identifiable: tuple.item,
+              identifiableInList: identifiableInList,
             );
             break;
           case ManageOperation.remove:
             yield ManagedList(
-              tuple.list._removeFromList(tuple.item),
+              tuple.list.removeFromList(tuple.item),
               identifiable: tuple.item,
               operation: ManageOperation.remove,
+              identifiableInList: identifiableInList,
             );
             break;
           case ManageOperation.ignore:
             yield ManagedList(
               tuple.list,
               identifiable: tuple.item,
               operation: ManageOperation.ignore,
+              identifiableInList: identifiableInList,
```

Comment:
What is the reason of moving `identifiableInList` from the callback to the `ManagedList` ?

Reply:
'identifiableInList' was moved so that it can be used in the dashboard service for determining the counter